capstone/capapi/templates/file_download.html

```diff
@@ -10,21 +10,28 @@
 
 {% block main_content %}
   {% if breadcrumbs %}
-  <nav aria-label="breadcrumb">
-    <ol class="breadcrumb">
-      {% for breadcrumb in breadcrumbs %}
-        {% if forloop.last %}
-          <li class="breadcrumb-item active" aria-current="page">{{ breadcrumb.name }}</li>
-        {% else %}
-          <li class="breadcrumb-item">
-            <a href="{% url 'download-files' %}/{{ breadcrumb.path }}">{{ breadcrumb.name }}</a>
-          </li>
-        {% endif %}
-      {% endfor %}
-    </ol>
-  </nav>
+    <nav aria-label="breadcrumb">
+      <ol class="breadcrumb">
+        {% for breadcrumb in breadcrumbs %}
+          {% if forloop.last %}
+            <li class="breadcrumb-item active" aria-current="page">{{ breadcrumb.name }}</li>
+          {% else %}
+            <li class="breadcrumb-item">
+              <a href="{% url 'download-files' %}/{{ breadcrumb.path }}">{{ breadcrumb.name }}</a>
+            </li>
+          {% endif %}
+        {% endfor %}
+      </ol>
+    </nav>
   {% endif %}
   <ul class="list-group">
+    {% if isroot %}
```

Comment:
It's a little weird to have this item not sorted alphabetically. It might actually simplify the view code a little to refactor a bit so manifest.csv doesn't need special treatment in the template: have the view put together a list of dicts like `[{"name":, "path":, "is_dir":, "size":}]`, sort by name, and then the view doesn't need a special file wrapper and the template doesn't need to know that the manifest.csv dict is special.

Reply:
But the manifest.csv will be a special case still since it won't be generated until called by the user, right? So we won't know how large the file is?

Reply:
Yep, true! I think that's fine ...